Commands/Commands/uservoice.js

```diff
@@ -225,7 +225,7 @@ commands.comment = {
       })
     }).catch(e => {
       if (e === 'Not found') {
-        msg.reply("I was unable to find your details, make sure you've logged into the website at least once.").then(errmsg => {
+        msg.reply("I was unable to find your details, make sure you've logged into the website at <https://`+ config.uservoice.subdomain + `.` + config.uservoice.domain + `> at least once.").then(errmsg => {
```

Comment:
Had all of these before, though Travis didn't like it... https://github.com/SteamingMutt/DiscordFeedback/pull/42/commits/d7393a36458ab7f1919ac2bd40e88dc0301c1160